src/services/autoLoginService.js

```diff
@@ -26,6 +27,7 @@ class AutoLoginService {
     const currentPath = history.location?.hash;
     const handleLoginFailure = () => {
       encryptionService.clearStoredAccount();
+      messagesService.clearLocalMessages();
```

Comment:
What I'm worried is we have this code

```
encryptionService.clearStoredAccount();
messagesService.clearLocalMessages();
```

in 4 or 5 diff places in app. So whenever we want to add another cleaning, we need to add it in all these places.
Maybe we can somehow optimise it.